extensions/sockets/src/main/java/org/mule/module/socket/internal/ConnectionEvent.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.socket.internal;
+
+import org.mule.module.socket.api.source.SocketAttributes;
+
+public class ConnectionEvent
```

Comment:
this class seems like a DTO of the MuleMessage. Why don't you just use MuleMessage?